src/beanmachine/ppl/diagnostics/tools/utils/accessor.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""Accessor definition for extending Bean Machine `MonteCarloSamples` objects."""
```

Comment:
Can you help me understand what is the purpose of this?

Reply:
I've updated the docstrings to include an explicit example for what this is doing.